jawr-spring/jawr-spring-boot-sample/pom.xml

```diff
@@ -63,13 +62,17 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
-			<version>${spring-boot-version}</version>
+			<version>${spring-boot.version}</version>
 		</dependency>
 		<!-- end::actuator[] -->
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-webmvc</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.github.dtrunk90</groupId>
+			<artifactId>thymeleaf-jawr-extension</artifactId>
+		</dependency>
```

Comment:
I guess you could add `<scope>provided</scope>` here. The auto configuration adds a Thymeleaf Dialect if the jar is in the projects classpath: `@ConditionalOnClass(name = "com.github.dtrunk90.thymeleaf.jawr.dialect.JawrDialect")`